contribs/gmf/options/gmfx.js

```diff
@@ -476,6 +477,14 @@ gmfx.AuthenticationFunctionalities.prototype.default_basemap;
  */
 gmfx.AuthenticationFunctionalities.prototype.default_theme;
 
+
+/**
+ * A list of layer names that can be filtered.
+ * @type {Array.<string>|undefined}
```

Comment:
By default an object can be null, for coherence live it like it, we will change it later if necessary :-)